data/entities.yaml

```
@@ -60,17 +60,26 @@
     attr: rock
     char: '@'
   description:
-  - A rock.
+  - A rock. Can be drilled to smaller rocks relatively fast.
```

I'd like to give a stronger hint that you can use a drill to get this out of the way.  Also, the description `A rock` should have been changed before when I changed the name.
```suggestion
  - A very large, impassable rock. A drill can be used to get rid of it and turn it into smaller rocks.
```